tests/integration_tests/functional/test_balloon.py

```diff
@@ -652,114 +599,69 @@ def _test_balloon_snapshot(context):
     microvm.kill()
```

Comment:
I guess we can get rid of this, right?

Reply:
We can yeah, as the microvm_factory will kill all VMs on teardown. Does not hurt though.